### ISSUE TITLE
How to configure it as a proxy server?

### ISSUE THREAD

**Issue by cs16lover** [jhao104/proxy_pool#234]
How can I configure a proxy server that reads from your proxy pool? And update proxy server settings by dynamically? (like update proxy list from proxy pool in every hour etc)

**Comment by 1again** [comment]
hello， it's a good question， but we not have the feature.

if you want this feature, i suggest you can reference [haiproxy](https://github.com/SpiderClub/haipproxy/blob/master/README_EN.md)

it's implement dynamically proxy server by squid as proxy server and python script as update squid proxy config file, maybe can help you!

**Comment by cs16lover** [comment]
> > How can I configure a proxy server that reads from your proxy pool? And update proxy server settings by dynamically? (like update proxy list from proxy pool in every hour etc)
> 
> we not provide proxy services, just code

Yes I know. I'm asking the code for proxy pool rotation? Does your software/code has that feature?
